code/Controllers/CMSMain.php

```diff
@@ -223,19 +233,23 @@ public function getResponseNegotiator(): PjaxResponseNegotiator
      * Get pages listing area
      *
      * @return DBHTMLText
+     * @deprecated 5.4.0 Will be renamed to RecordList
      */
     public function PageList()
     {
+        Deprecation::noticeWithNoReplacment('5.4.0', 'Will be renamed to RecordList');
         return $this->renderWith($this->getTemplatesWithSuffix('_PageList'));
     }
 
     /**
      * Page list view for edit-form
      *
      * @return DBHTMLText
+     * @deprecated 5.4.0 Will be removed without equivalent functionality
      */
     public function PageListSidebar()
     {
+        Deprecation::noticeWithNoReplacment('5.4.0');
         return $this->renderWith($this->getTemplatesWithSuffix('_PageList_Sidebar'));
```

Comment:
It'd just a template that redirects to a different template. Completely pointless.